src/api/app/models/bs_request.rb

```diff
@@ -781,15 +797,7 @@ def addreview(opts)
       self.commenter = User.current.login
       self.comment = opts[:comment] if opts[:comment]
 
-      newreview = reviews.create(
-        reason:     opts[:comment],
-        by_user:    opts[:by_user],
-        by_group:   opts[:by_group],
-        by_project: opts[:by_project],
-        by_package: opts[:by_package],
-        creator:    User.current.try(:login),
-        reviewer:    User.current.try(:login)
-      )
+      newreview = _create_new_review(opts)
```

Comment:
why with underscore? :confused:

Reply:
just to see if the test suite catches it